### PR TITLE
Make code more legible by aligning columns, comments, etc (CCExtractor Development) 

### PR DESCRIPTION
**Pull request by ngxingyu**
google code-in task - worked on the .h files in  src/lib_ccx/

### CODE REVIEW

File: src/lib_ccx/asf_constants.h [hunk 1]
```diff
@@ -21,74 +21,108 @@
...
 	int AudioStreamNumber;
 	int CaptionStreamNumber;
-	int CaptionStreamStyle; // 1 = NTSC, 2 = ATSC
-	int DecodeStreamNumber; // The stream that is chosen to be decoded
-	int DecodeStreamPTS; // This will be used for the next returned block
-	int currDecodeStreamPTS; // Time of the data returned by the function
-	int prevDecodeStreamPTS; // Previous time
-	int VideoStreamMS; // See ableve, just for video
+	int CaptionStreamStyle; 
+// 1 = NTSC, 2 = ATSC
```
Comment (canihavesomecoffee) [inline src/lib_ccx/asf_constants.h]: This does not make it more readable.

File: src/lib_ccx/asf_constants.h [hunk 2]
```diff
@@ -21,74 +21,108 @@
...
-	// Store the Payload Extension System Data Size.  First index holds the
-	// stream number and the second index holds the Extension System entry.
-	// I.e. PayloadExtSize[1][2] is the third Payload Extension System
-	// entry for stream 1. (The streams are numbered starting from 1)
-	// FIXME: What happens if we have more than 9 streams with more than
-	// 10 entries.
+	
+// Extended Stream Properties  - for DVR-MS presentation timestamp
+	
+// Store the Payload Extension System Data Size.  First index holds the
```
Comment (canihavesomecoffee) [inline src/lib_ccx/asf_constants.h]: Neither do these changes.

File: src/lib_ccx/stdintmsc.h [hunk 3]
```diff
@@ -27,21 +28,47 @@
 // OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
 // ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
-///////////////////////////////////////////////////////////////////////////////
+//
```
Comment (canihavesomecoffee) [inline src/lib_ccx/stdintmsc.h]: What is this supposed to improve in readability?

**Comment by ngxingyu** [comment]
modified it

**Comment by cfsmp3** [comment]
Comments still not aligned
https://github.com/CCExtractor/ccextractor/pull/542/files

Plus cannot be merged anyway, conflicts everywhere

**Pull request closed without merging**